amplify/backend/function/risestandstrongcb34053a/src/routes/shifts.js

```diff
@@ -2,6 +2,7 @@ const express = require('express');
 const router = express.Router();
 const { postShift, queryShiftsRange } = require('../utils/aws-utils');
 const { getShift } = require('../utils/aws-utils');
+const { putShift } = require('../utils/aws-utils');
```

Comment:
I think it's a bit more clear to just use one require statement here. You can combine these by doing `const { postShift, queryShiftsRange, getShift, putShift } = require('../utils/aws-utils');`